src/js/api-client.js

```diff
@@ -0,0 +1,42 @@
+import * as ApiConfig from './config';
+
+export const getCurrentWeather = async (location) => {
+  if (!location) throw Error('please provide a valid location');
+  const response = await fetch(`${ApiConfig.apiUrl}?q=${location}&appid=${ApiConfig.apiKey}`);
+  const { status } = response;
+  switch (status) {
+    case 200:
+    {
+      const json = await response.json();
+      const { main, description, icon } = json.weather[0];
+      const { main: { humidity, pressure, temp } } = json;
+      return {
+        humidity, pressure, temp, main, description, icon,
+      };
+    }
+
+    default:
+    {
+      const { cod, message } = await response.json();
+      const msg = message.replace(/city/, `The city ${location.toLowerCase()} was`);
+      throw Error(`${cod}: ${msg}`);
+    }
+  }
+};
+
+export const convertTemperature = (tempInKelvin, tempFormat) => {
```

Comment:
There is suppose to be a request made for both Celsius and Fahrenheit data you aren't supposed to get one and calculate the other ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png)